packages/cmf/src/sagas/http.js

```diff
@@ -49,11 +49,14 @@ export class HTTPError extends Error {
 export function handleBody(response) {
 	let methodBody = 'text';
 
-	const contentType = response.headers.get('Content-Type');
+	if (response.headers) {
```

Comment:
get content-type with lodash get, provide a default value and avoid the unnecessary if statements